js/tabler.js

```diff
@@ -112,6 +112,13 @@ $(document).ready(function() {
     e.preventDefault();
     return false;
   });
+  
+  $(".custom-file-input").on('change',function() { 
+	      // Get the complete path and keep only the filename
+        var filename = $(this).val().split('\\').pop();
```

Comment:
Please use `const` as you do not modify this variable.

Reply:
Also instead of `pop()`  use `.slice(0, -1)` as it does not modify source array.